### DISCUSSION TITLE
The database file was locked when the orientdb cluster was created on k8s

### DISCUSSION THREAD

**Discussion by xiashengjie** [orientechnologies/orientdb#10009] [Q&A]
I am using fluid and juicefilesystem on k8s to separate storage and storage of distributed orientdb cluster. Now three databases of orientdb Pods are using the same PVC. After starting one pod, the remaining pods cannot be started, indicating that the database files are locked. 
How should I configure to be able to use the same PVC?

**Comment by xiashengjie** [comment]
My yaml file is as follows
apiVersion: apps/v1
kind: StatefulSet
metadata:
  name: orientdb 
  namespace: default
  labels:
    app: orientdb 
spec:
  selector:
    matchLabels:
      app: orientdb 
  replicas: 3
  serviceName: orientdb 
  template:
    metadata:
      labels:
        app: orientdb 
        orientdb-cluster-member: "true"
    spec:
      serviceAccountName: hazelcast-k8s
      terminationGracePeriodSeconds: 30
      containers:
        - name: orientdb
          command: ["sh", "-c", "rm /dev/random; mknod /dev/random c 1 9; dserver.sh -DkubernetesNamespace=default;cp -r /opt/fluid_jfs_orientdb/orientdbconfig/* /orientdb/config/"]
          image: orientdb:3.2.15
          imagePullPolicy: Always
          env:
            - name: ORIENTDB_NODE_NAME
              value: orientdb 
          ports:
            - containerPort: 2480
              protocol: TCP
            - containerPort: 2424
              protocol: TCP
            - containerPort: 2434
              protocol: TCP
          livenessProbe:
            tcpSocket:
              port: 2424
            initialDelaySeconds: 15
            periodSeconds: 20
            failureThreshold: 5
          readinessProbe:
            tcpSocket:
              port: 2424
            initialDelaySeconds: 10
            periodSeconds: 20
            failureThreshold: 5
          volumeMounts:
          - name: jfs-orientdb
            mountPath: /orientdb/databases
          - name: config
            mountPath: /orientdb/config
      initContainers:
      - name: copy-configs
        image: busybox:1.31
        command: ['sh', '-c']
        args:
          - cp -r /opt/fluid_jfs_orientdb/orientdbconfig/* /orientdb/config/
        volumeMounts:
          - name: config
            mountPath: /orientdb/config/
          - name: orientdb-config-cm-vol
            mountPath: /opt/fluid_jfs_orientdb/orientdbconfig/
      volumes:
      - name: jfs-orientdb
        persistentVolumeClaim:
          claimName: jfs-orientdb
      - name: config
        emptyDir: {}
      - name: orientdb-config-cm-vol
        hostPath:
          path: /opt/fluid_jfs_orientdb/orientdbconfig/

**Comment by andrii0lomakin** [comment]
Hi, 
You can not use several instances of DB running inside the same file path. Is that what you are doing? I am not a k8s expert, unfortunately.
If your file systems,in reality, are separated, then you can disable file locking by providing a flag.
Please let me know if you need detailed instructions on how to set up this flag.
